README.md

```diff
@@ -1,117 +1,125 @@
 
-# pnpm
-pnpm i clerk-sveltekit
+```shell
+pnpm add clerk-sveltekit
+```
 
-# yarn
+```shell
 yarn add clerk-sveltekit
+```
 
-# bun
-bun i clerk-sveltekit
+```shell
+bun add clerk-sveltekit
 ```
 
+## Usage
+
 ### Set up environment variables
 
-Add these values to your `.env` (get them from Clerk after creating an application there):
+If you haven't already, create a `.env` file at the root of your application. Inside, add these two environment variables:
 
 ```env
-PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_abcdefg123
-CLERK_SECRET_KEY=sk_test_abcdefg123
+PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_xxx
+CLERK_SECRET_KEY=sk_test_xxx
 ```
 
-The easiest way to get these values is to click "API Keys" in the Clerk dashboard, and then copy the values for Next.js, and change `NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY` to `PUBLIC_CLERK_PUBLISHABLE_KEY`.
+These keys can always be retrieved from the [API keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard. Inside the dropdown choose "Astro" and copy/paste the values.
 
-Note that for production sites using OAuth providers, you will have to do some more setup with Clerk and DNS.
+**Note:** For production instances using OAuth providers, you will have to do some more setup with Clerk and DNS.
 
 ### Configure the server hook
 
 Add this to `src/hooks.server.ts` (or integrate this code with your existing `hooks.server.ts` file):
 
-```typescript
-import type { Handle } from '@sveltejs/kit'
-import { sequence } from '@sveltejs/kit/hooks'
-import { handleClerk } from 'clerk-sveltekit/server'
-import { CLERK_SECRET_KEY } from '$env/static/private'
-
-export const handle: Handle = sequence(
-	handleClerk(CLERK_SECRET_KEY, {
-		debug: true,
-		protectedPaths: ['/admin'],
-		signInUrl: '/sign-in',
-	})
-)
+```ts
+import { withClerkHandler } from 'clerk-sveltekit/server'
+
+export const handle = withClerkHandler()
+```
+
+Update the `app.d.ts` file to ensure that the locals added by the Clerk handler are properly typed.
+
+```ts
+/// <reference types="clerk-sveltekit/env" />
+
+declare global {
+	namespace App {...}
+}
 ```
 
-### Configure the client hook
+### Add <ClerkProvider> to your root layout
+
+All Clerk components must be children of the `<ClerkProvider>` component, which provides active session and user context.
 
 Add this to `src/hooks.client.ts`:
 
-```typescript
-import type { HandleClientError } from '@sveltejs/kit'
-// To use Clerk components:
-import { initializeClerkClient } from 'clerk-sveltekit/client'
-// Or for headless mode:
-// import { initializeClerkClient } from 'clerk-sveltekit/headless'
-import { PUBLIC_CLERK_PUBLISHABLE_KEY } from '$env/static/public'
-
-initializeClerkClient(PUBLIC_CLERK_PUBLISHABLE_KEY, {
-	afterSignInUrl: '/admin/',
-	afterSignUpUrl: '/admin/',
-	signInUrl: '/sign-in',
-	signUpUrl: '/sign-up',
-})
-
-export const handleError: HandleClientError = async ({ error, event }) => {
-	console.error(error, event)
+```ts
+// src/+layout.server.ts
```

Comment:
Is this correct? Right above you say to add to src/hooks.client.ts

Reply:
Nice catch! Updated